src/components/search/related/Related.js

```diff
@@ -68,13 +65,7 @@ export function Related({ data, isLoading }) {
       space={{
         bottom: "var(--pt4)",
       }}
-      title={
-        noRelatedSubjects ? (
-          <span />
-        ) : (
-          <FilterButton isLoading={isLoading} className={styles.filterButton} />
-        )
-      }
+      title={<span />}
```

Comment:
Skal det være en title ? kan den være null?

Reply:
hvis man ikke giver en title, så får man default titlen. til gengæld tjekker vi i Section komponenten en del gange, hvis vi har en title på trods af at den ikke kan være null/undefined pga default værdien... 

så måske kunne man fjerne default værdien fra Section komponenten...

Reply:
Ja lyder som at default værdien burde fjernes. Men det har så ikke noget med det her pull at gøre. Så helt fint for mig at beholde <span /> for nu, og så kan vi undersøge refactor senere.